Week3/homework/step4.js

```diff
@@ -1,8 +1,14 @@
 'use strict';
 
 // Add your code here
+function createBase(base) {
+  function addToBase(value) {
+    return base + value;
+  }
+  return addToBase;
 
-const addSix = createBase(6);
+  const addSix = createBase(6);
 
-addSix(10); // returns 16
-addSix(21); // returns 27
+  addSix(10); // returns 16
+  addSix(21); // returns 27
+}
```

Comment:
Again, it is not tested. :) The idea is really good, though! Just that you should have closed the createBase function after _return addToBase;_ 
If you really understood the idea, it is great! But please keep in mind to test, closing the function body in a strange way, after calling those function might give the impression that you didn't really understand the logic behind, which I hope it is not the case. :)